src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -157,6 +167,88 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
             ElementBinder.SetElementForTrace(InternalElement);
         }
 
+        private void InitFamilyInstance(FamilySymbol fs, Reference reference, Line pos)
+        {
+            //Phase 1 - Check to see if the object exists and should be rebound
+            var oldFam =
+                ElementBinder.GetElementFromTrace<Autodesk.Revit.DB.FamilyInstance>(Document);
+
+            //There was a point, rebind to that, and adjust its position
+            if (oldFam != null)
+            {
+                InternalSetFamilyInstance(oldFam);
+                InternalSetFamilySymbol(fs);
+                InternalSetFace(reference);
+                InternalSetPosition(pos);
+                return;
+            }
+
+            //Phase 2- There was no existing point, create one
+            TransactionManager.Instance.EnsureInTransaction(Document);
+
+            //If the symbol is not active, then activate it
+            if (!fs.IsActive)
+                fs.Activate();
+
+            Autodesk.Revit.DB.FamilyInstance fi;
+
+            if (Document.IsFamilyDocument)
+            {
+                fi = Document.FamilyCreate.NewFamilyInstance(reference, pos, fs);
+            }
+            else
+            {
+                fi = Document.Create.NewFamilyInstance(reference, pos, fs);
+            }
+
+            InternalSetFamilyInstance(fi);
+
+            TransactionManager.Instance.TransactionTaskDone();
+
+            ElementBinder.SetElementForTrace(InternalElement);
+        }
+
+        private void InitFamilyInstance(FamilySymbol fs, Reference reference, XYZ location,
+            XYZ referenceDirection)
```

Comment:
I'm not sure how to set the reference direction on the internal family instance in the case of element binding?